### ISSUE TITLE
Default answers to LLM or spoken to be editable

### ISSUE THREAD

**Issue by WhiskerWeirdo** [WhiskerWeirdo/BanterBrain-Buddy#66]
I want BBB to allow the default responses send to LLM or being spoken to be editable. They are now hardcoded.

**Comment by WhiskerWeirdo** [comment]
This is being added in 1.0.3 I added the initial base for this using the English language. 
```{
  "Language": "English",
  "CheerTalkToLLM": "respond to the message of {user} who said: {message}",
  "CheerDefaultNoMessage": "thank you for the cheer, next time submit a message!",
  "CheerWithMessage": "{user} cheered with message {message}",

  "ChannelPointTalkToLLM": "Respond to the message of {user} saying: {message}",
  "ChannelPointDefaultNoMessage": "Thank you for the redeem, next time submit a message!",
  "ChannelPointWithMessage": "{user} redeemed with message {message}",

  "GiftedSingleSub": "Thanks {user} for gifting {amount} tier {tier} sub!",
  "GiftedMultipleSubs": "Thanks {user} for gifting {amount} tier {tier} subs!",

  "SubscribeFirstTimeThanks": "Thanks {user} for subscribing!",
  "SubscribeResubThanksWithMessageLLM": "Respond to the message of {user} saying: {message}",

  "SubscribeMonthsNoMessage": "{user} has resubscribed for a total of months {months}!",
  "SubscribeMonthsMessage": "{user} has resubscribed for a total of {months} months saying {message}.",

  "ChatMessageResponseLLM": "respond to {user} who said: {message}",
  "ChatMessageUserSaid": "{user} said {message}"
}```

**Comment by WhiskerWeirdo** [comment]
fixed in 1.0.6
